.gitignore

```diff
@@ -11,7 +11,7 @@ __diff_output__
 /.nyc_output
 /coverage
 /docs/.next
-/docs/pages/playground*
+/docs/pages/playground/
```

Comment:
This was specifically done like that to account for people already used to using `playground<some_text>.tsx`.
This PR would sort of break your current use-case @flaviendelangle , wouldn't it?

Reply:
Yes
But as we discussed, I can stop using `playground.tsx`
We would just to be careful not to push any old `playground.tsx` by mistake on our PRs in the coming days.